src/components/DirectoryTree.tsx

```diff
@@ -281,24 +281,27 @@ export class DirectoryTree extends React.Component<DirectoryTreeProps, {
        * dropped into target or some parent of the target.
        */
       onDragOver(tree: ITree, data: IDragAndDropData, targetElement: File, originalEvent: DragMouseEvent): IDragOverReaction {
-        const items = Array.from(originalEvent.browserEvent.dataTransfer.items);
-        function mimeTypeUploadIsAllowed(type: string) {
-          console.log(type);
-          if (type === "") { // Firefox doesn't show the "application/wasm" mime type.
-            return true;
+        // File being dragged into the browser
+        if (!(data as any).elements) {
+          const items = Array.from(originalEvent.browserEvent.dataTransfer.items);
+          function mimeTypeUploadIsAllowed(type: string) {
```

Comment:
Normally (utility) function definition (without closure) needs to be done outside other function/class definition. Can you move it outside class method. Can we also fix bad name to be in form of verb-subject, e.g. `isUploadAllowedForMimeType(type: string)`?